sygnal/apnstruncate.py

```diff
@@ -85,7 +86,7 @@ def truncate(payload, max_length=2048):
 
 
 def _choppables_for_aps(aps):
-    ret = []
+    ret: List[Union[Tuple[str], Tuple[str, int]]] = []
```

Comment:
I think we've usually be using the `# type: Foo` style in Synapse, I believe this is due to Python 3.5 not supporting this syntax? I don't know if that's supported for Syngal.

Also 👍 on adding this!

Reply:
Since the start of the Sygnal rewrite, the readme has said we require Python 3.7+, so should be good :)